metricbeat/module/php_fpm/_meta/config.yml

```diff
@@ -0,0 +1,6 @@
+- module: php_fpm
```

Comment:
The module I started looking at to learn is `haproxy`, which is no commented out. But I'll do it then!